pkg/auth/providers/activedirectory/activedirectory_actions.go

```diff
@@ -98,7 +109,14 @@ func (p *adProvider) saveActiveDirectoryConfig(config *v3.ActiveDirectoryConfig)
 	config.Type = client.ActiveDirectoryConfigType
 	config.ObjectMeta = storedConfig.ObjectMeta
 
-	logrus.Debugf("updating githubConfig")
+	name := fmt.Sprintf("%s:%s-%s", "mgmt-secrets", strings.ToLower(convert.ToString(config.Type)), "serviceaccountpassword")
+	if err := common.CreateOrUpdateSecrets(p.secrets, config.ServiceAccountPassword, "serviceaccountpassword", strings.ToLower(convert.ToString(config.Type))); err != nil {
+		return err
+	}
+
+	config.ServiceAccountPassword = name
+
+	logrus.Debugf("updating activeDirectoryConfig")
```

Comment:
Are we planning to leave debug logs in? do we want to make them more descriptive (omitting sensitive details of course_=)

Reply:
Authconfigs already had their own debug logs actually for each action, I didn't add them 

This is just an edit for AD where by mistake, it said githubConfig